docs/administration-guide/gaffer-stores/federated-store.md

```diff
@@ -2,4 +2,405 @@
 
 The Federated Store is a Gaffer store which forwards operations to a collection of sub-graphs and returns a single response as though a single graph were queried.
 
-For more information on the federated store, see [the v1 federated store walkthrough](https://gchq.github.io/gaffer-doc/v1docs/getting-started/developer-guide/federatedstore.html) (to be updated and added to this documentation in future).
\ No newline at end of file
+## Introduction
+
+The Federated Store facilitates querying of multiple Gaffer graphs through a single endpoint/instance, and the federation of queries across many graphs at once - with results automatically merged.
+Unlike the Map or Accumulo Store, the Federated Store doesn't use a schema of its own. This is because it represents a graph of graphs.
+
+The Federated Store graph is a collection of other graphs. These sub-graphs can be any kind of Gaffer graph, including further federated graphs.
```

Comment:
This is a better way of describing it, a collection of graphs to which queries are federated to, rather than a graph of graphs.